test/Keywords.test.kt

```diff
@@ -69,4 +69,18 @@
 //       ^ source.kotlin
 //        ^ source.kotlin meta.block.kotlin punctuation.section.group.begin.kotlin
   }
-//^ source.kotlin meta.block.kotlin punctuation.section.group.end.kotlin
\ No newline at end of file
+//^ source.kotlin meta.block.kotlin punctuation.section.group.end.kotlin
+
+  val test = !isC
```

Comment:
I would also add a regression test for this issue for additional coverage. see the `regression` folder for details on how to do that.